packages/next-codemod/transforms/__testfixtures__/update-router-api/use-router-push.output.js

```diff
@@ -0,0 +1,22 @@
+import { useRouter } from 'next/router'
+
+function ActiveLink({ children, href, as }) {
+  const router = useRouter()
+  const style = {
+    marginRight: 10,
+    color: router.pathname === href ? 'red' : 'black',
+  }
+
+  const handleClick = (e) => {
+    e.preventDefault()
+    router.push(href)
```

Comment:
https://github.com/vercel/next.js/pull/20262/files#r545058425

```suggestion
    router.push(as)
```